### ISSUE TITLE
More read-only notebook previews

### ISSUE THREAD

**Issue by zarsky-broad** [DataBiosphere/saturn-ui#1085]
If a user's cluster is stopped or non-existent, show a ready-only preview of a notebook when it's clicked, with a small top bar explaining that it's read-only and a progress indicator when resuming. Editable notebook should be loaded once the cluster is in 'running' state

**Comment by katevoss** [comment]
AoU's mockup design:
<img width="1241" alt="screen shot 2019-01-22 at 2 23 41 pm" src="https://user-images.githubusercontent.com/2395211/51560047-7a9ffb80-1e51-11e9-9be3-722e3210ed78.png">

**Comment by katevoss** [comment]
AC: 
- [ ] with no cluster, I see a read-only version of a notebook
- [ ] with a cluster that is creating, I see a read-only version of a notebook
- [ ] with a cluster that is running while viewing a read-only version of a notebook, I see a link and message that gives the option of switching to edit 
- [ ] with Owner/Writer access, if Leo does not respond then I see a read-only version of a notebook (from #1003)
- [ ] with Owner/Writer access, I can open a read-only version of a notebook (from #1070) when opening the notebook, and I see a link and message that gives the option of switching to edit

**Comment by mbookman** [comment]
Should we bring forward the labels from the consolidated issues?
#1070 had 2019-March  Beta-response  amp-pd

**Comment by katevoss** [comment]
Splitting this into multiple PRs, the first of which is here: https://github.com/DataBiosphere/saturn-ui/pull/1239

**Comment by katevoss** [comment]
Next PR: #1251

**Comment by mbookman** [comment]
Please provide a comment as to why this is closed.
There is a PR out, but it has not been merged.
If the intent is that the issue can be tracked elsewhere, please state that explicitly.

Thanks!

**Comment by bradtaylor** [comment]
@mbookman I think there's some confusion going on among a set of related issues here. The present issue is about adding in read-only preview to provide the user with some ability to review the notebook during that unfortunate time span where their VM is creating or un-pausing. This has been merged and released to production.

Your related issue #1070 requests on-demand read-only preview. We are doing that. It is in review with a separate PR https://github.com/DataBiosphere/saturn-ui/pull/1239. So 

So I believe the two issues are in the correct respective states. The genesis of the present ticket was as part of our response to removing cluster auto-create. I therefore aded the "internal-ideas label". #1070 will close once the open PR is merged, reviewed and released. It carries the "Beta-response" and "amp-pd" labels, the the request originated with you.

I have a to-do to reach out to you and Nicole with some questions about some particular aspects of the UX around read-only previews.

I think it would be good to have a standing checkin between you me and Nicole to review the status of your top issues so that we can talk about implementation timelines and challenges and hopefully avoid confusion. Would you be interested in that?

Thanks!

**Comment by mbookman** [comment]
Thanks Brad!

**Comment by mbookman** [comment]
BTW, the features added by @katevoss in #1239 are *awesome*. The read-only view while waiting for my notebook cluster to start is a real time saver!
